geocity/apps/core/static/js/submission_submit.js

```diff
@@ -8,3 +8,9 @@ checkbox.addEventListener("click", () => {
         button.toggleAttribute("disabled");
     }
 });
+
+let formNumber = document.querySelector("[data-geo-time-role='formNumber']");
+
+if (formNumber.textContent == "1") {
```

Comment:
Count number of elements instead of just hiding the "1".
Put in a function and add event listener on the button used to add new geotimes

Reply:
@AlexandreJunod In fact I did the same logic as you did:
1. https://github.com/yverdon/geocity/blob/main/geocity/apps/core/static/js/submissions_submit.js#L3
2. https://github.com/yverdon/geocity/blob/main/geocity/apps/core/static/js/submissions_geo_time.js#L13